[JDBC] Gradle build fails due to the common/JDBC module.

`./gradlew build` results in an error, due to the `common/jdbc` module:

```
> Task :common:jdbc:compileScala FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':common:jdbc:compileScala
> java.lang.NullPointerException (no error message)
```

Additionally, the version of `zinc` is `2.10.5`, while the major version of Scala being brought in is `2.11`.  Mixing major versions of scala packages often results in issues (plus zinc has been deprecated for a while) - this may or may not be related, I haven't done any digging for the issue above.

--- a/common/jdbc/build.gradle
+++ b/common/jdbc/build.gradle
@@ -16,9 +16,6 @@ dependencies {
     implementation group: 'org.apache.spark', name: 'spark-mllib_2.11'
     implementation group: 'com.typesafe', name:'config'
 
-    zinc group: 'com.typesafe.zinc', name: 'zinc', version: '0.3.9'
-    zinc group: 'org.scala-lang', name: 'scala-library', version: '2.10.5'
-
     testCompile group: 'org.scalatest', name: 'scalatest_2.11', version: '3.0.5'
     testRuntimeOnly group:'org.scala-lang.modules', name: 'scala-xml_2.11', version: '1.1.1'
     testImplementation group: 'com.google.truth.extensions', name: 'truth-java8-extension'

